crates/bifrost/src/providers/replicated_loglet/network.rs

```diff
@@ -108,12 +112,53 @@ impl RequestPump {
                 Some(append) = self.append_stream.next() => {
                     self.handle_append(&provider, append).await;
                 }
+                Some(get_tail_info) = self.get_tail_info_stream.next() => {
+                    self.handle_get_tail_info(&provider, get_tail_info).await;
+                }
             }
         }
 
         Ok(())
     }
 
+    async fn handle_get_tail_info<T: TransportConnect>(
+        &mut self,
+        provider: &ReplicatedLogletProvider<T>,
+        incoming: Incoming<GetTailInfo>,
+    ) {
+        let (reciprocal, body) = incoming.split();
+
+        let loglet = match self.get_loglet(provider, &body.header).await {
+            Ok(loglet) => loglet,
+            Err(err) => {
+                return_error_status!(reciprocal, err);
+            }
+        };
+
+        if !loglet.is_sequencer_local() {
+            return_error_status!(reciprocal, SequencerStatus::NotSequencer);
+        }
+
+        let tail = loglet.known_global_tail().get();
+        let tail_info = TailInfo {
+            header: CommonResponseHeader {
+                known_global_tail: Some(tail.offset()),
+                sealed: Some(tail.is_sealed()),
+                status: if tail.is_sealed() {
+                    SequencerStatus::Sealed
+                } else {
+                    SequencerStatus::Ok
+                },
+            },
+        };
+
+        let _ =
+            task_center().spawn_child(TaskKind::Disposable, "return-tail-info", None, async move {
+                reciprocal.prepare(tail_info).send().await?;
+                Ok(())
+            });
+    }
```

Comment:
Can we use try_send() instead and not create a task? It's okay to fail if the channel is full

Reply:
Sure

Reply:
I am wondering if we should use try_send() in the error handling macro as well ?